sshnp_docker_demo/db-template.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+#
+# DO NOT RUN THIS SCRIPT. THIS IS MEANT TO BE RUN BY ANOTHER SCRIPT IN ONE OF THE THREE DIRECTORIES
+#
+
+if [ -z "$1" ]
+then
+    echo "Usage: ./db.sh <sshnp/sshnpd/sshrvd>"
+    exit 1
+fi
+
+rm Dockerfile
```

Comment:
Test for the file first before trying to delete it to avoid an error message

Reply:
instead, i thought it was a good idea to copy overwrite/force - https://github.com/atsign-foundation/at_demos/pull/97/commits/57a70a1e7227d0dae77eb2703181faa1b6b60977

for the second `rm Dockerfile`, i do the if check - https://github.com/atsign-foundation/at_demos/pull/97/commits/a12d9836483c3e2fa38e65a2fdec37c1e70bb738